backend/setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = tracim_backend
-version = 2.7.4
+version = 2.7.5
```

Comment:
This makes me think that we'll have to change this version to `3.0.0`, would you think this can be done in this PR ?

Reply:
This PR is just a cherry pick of old commit so, it make sense to set version to 2.7.5 here.
Bump to 3.0.0 has to be done in https://github.com/tracim/tracim/issues/2785